static/js/message_util.js

```diff
@@ -23,7 +23,7 @@ exports.message_has_link = function (message) {
 };
 
 exports.message_has_image = function (message) {
-    return $(message.content).find(".message_inline_image").length > 0;
+    return $(message.content).hasClass("message_inline_image");
```

Comment:
I think it's possible for `message_inline_image` elements to be in a descendant, e.g. using spoilers or block quotes.  Do we need to do both checks, or is there a better way to express this?

Reply:
@timabbott I am sending a PR in a moment regarding my fix.

Reply:
Ohh, Yes for spoiler it can be descendant. I am looking for a better way to express this other than doing both the checks.